lib/plug/conn.ex

```diff
@@ -655,6 +655,97 @@ defmodule Plug.Conn do
     for {^key, value} <- headers, do: value
   end
 
+  @doc ~S"""
+  Prepends the list of headers to the connection request headers.
+
+  Similar to `put_req_header` this functions adds a new request header
+  (`key`) but rather than replacing the existing one it prepends another
+  header with the same `key`.
+
+  The "host" header will be overridden by `conn.host` and should not be set
+  with this method. Instead, do `%Plug.Conn{conn | host: value}`.
+
+  Because header keys are case-insensitive in both HTTP/1.1 and HTTP/2,
+  it is recommended for header keys to be in lowercase, to avoid sending
+  duplicate keys in a request.
+  Additionally, reqonses with mixed-case headers served over HTTP/2 are not
```

Comment:
Please do a grep for `reqonses`, it happens in a couple places. :)

Reply:
Oops. Fixed.